openstack_controller/tests/metrics.py

```diff
@@ -4541,7 +4541,10 @@
     {
         'name': 'openstack.glance.image.count',
         'count': 1,
-        'value': 1,
+        'value': 1.0,
+        'metric_type': 0,
```

Comment:
These values, I am afraid, should not be here.

Reply:
I initially included a misformatted image test file, which seems to have required changing the value of value, as well as asserting the "metric_type" and "flush_first_value" parameter in order to pass pre-existing test cases. However, in addressing Sarah's comment through commit [54b8ce0](https://github.com/DataDog/integrations-core/pull/16976/commits/54b8ce0a2e1f0621486639ddd1839855d1265e3a), these changes were now unnecessary, so I have undone them.